doc/data/messages/r/redundant-unittest-assert/good.py

```diff
@@ -3,5 +3,9 @@
 
 class DummyTestCase(unittest.TestCase):
     def test_dummy(self):
-        actual = "test_result"
-        self.assertEqual(actual, "expected")
+        # Nothing, as an assert of a string literal will always pass
```

Comment:
I actually kind of like the current example. I think the comment works nicely in `details.rst`.